tests/resources/VCH-Util.robot

```diff
@@ -64,7 +64,7 @@ Get Docker Params
     Remove Environment Variable  DOCKER_HOST  DOCKER_TLS_VERIFY  DOCKER_CERT_PATH
 
     # Split the log log into pieces, discarding the initial log decoration, and assign to env vars
-    ${logmon}  ${logday}  ${logyear}  ${logtime}  ${loglevel}  ${vars}=  Split String From Right  ${line}  ${SPACE}  5
+    ${logmon}  ${logday}  ${logyear}  ${logtime}  ${loglevel}  ${vars}=  Split String  ${line}  ${SPACE}  5
```

Comment:
`Group11-Upgrade` tests are failing in build 7667, but I'm not sure if those failures are related to this.